codegen/src/main/java/software/amazon/awssdk/codegen/poet/builder/AsyncClientBuilderClass.java

```diff
@@ -32,6 +32,7 @@ public class AsyncClientBuilderClass implements ClassSpec {
     private final ClassName builderInterfaceName;
     private final ClassName builderClassName;
     private final ClassName builderBaseClassName;
+    // private final EndpointRulesSpecUtils endpointRulesSpecUtils;
```

Comment:
Remove commend-out lines?

Reply:
See above comments, but I just commented these out to make the split easier. They'll back in the next PR so hope you're fine just keeping them here until then